src/Users/EloquentUser.php

```diff
@@ -220,6 +220,25 @@ public function inRole($role)
         return false;
     }
 
+    /**
+     * @param $role
+     * @return bool
+     */
+    public function inAnyRole($roles)
+    {
+        if (is_array($roles) && !empty($roles)) {
+            foreach ($roles as $role) {
+                if ($this->inRole($role)) {
+                    return true;
+                }
+            }
+        } else {
+            return $this->inRole($roles);
+        }
+
+        return false;
```

Comment:
Just wondering if this should move to line `234` instead? As it seems this would only be hit if the loop ends without any results?

Reply:
I was keeping to the same structure as the [inRole](https://github.com/cartalyst/sentinel/blob/36d802f962dc14f7c0639a94976892fa0708dc23/src/Users/EloquentUser.php#L220) method.

Reply:
It's all good, no worries.

Will merge tomorrow and tag it, thanks!

Reply:
So I was also just copying the code from the issue linked. From what I understand changing where you return from in this situation is opinion based. I'm happy to change it. I'm just wondering if theres any more reasoning behind keeping the return within the ```if``` statement and keeping it at the end of the method.

Reply:
It's all good @8633brown :)